release-notes-agent/src/main/java/com/epam/release/notes/agent/service/action/EmailSendActionNotificationService.java

```diff
@@ -36,11 +37,14 @@
 
     private final JavaMailSender javaMailSender;
     private final TemplateNotificationService templateNotificationService;
+    private final String mailSender;
 
     public EmailSendActionNotificationService(final JavaMailSender javaMailSender,
-                                              final TemplateNotificationService templateNotificationService) {
+                                              final TemplateNotificationService templateNotificationService,
+                                              @Value("${spring.mail.username}") final String mailSender) {
```

Comment:
lets rename it somehow, because it is very similar to name `javaMailSender`
maybe `notificationServiceAccount` or `serviceEmail`